ui/src/literature/components/LiteratureDate/LiteratureDate.jsx

```diff
@@ -1,23 +1,28 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 
+import './LiteratureDate.scss';
+
 class LiteratureDate extends Component {
   render() {
-    const { date } = this.props;
+    const { date, wrapperClassName } = this.props;
 
     return (
-      <span>{date}</span>
+      <span className="__LiteratureDate__">
+        <span className={wrapperClassName}>{date}</span>
```

Comment:
this is confusing to push into this component, just to display it with parenthesis in some places. You could just wrap the component in jsx markup with parenthesis wherever you want.